### PR TITLE
Third PR Readme Updates for AAD To Microsoft Entra ID

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/tab-channel-group-config-page-auth/csharp/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 page_type: sample
-description: This sample demonstrates AAD and Silent Authentication inside teams configurable tab.
+description: This sample demonstrates Microsoft Entra and Silent Authentication inside teams configurable tab.
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/tab-channel-group-config-page-auth/csharp/README.md:3]: fixed

File: samples/tab-adaptive-cards/csharp/README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 - In the Azure Portal, navigate back to the Azure Bot resource created (https://learn.microsoft.com/microsoftteams/platform/bots/how-to/authentication/add-authentication?tabs=dotnet%2Cdotnet-sample#azure-ad-v2)
   -Switch to the "Settings" blade and click "Add Setting" under the OAuth Connection Settings section
   - Enter a name for your new Connection setting.  
-    - In the Service Provider dropdown, select Azure Active Directory V2
+    - In the Service Provider dropdown, select Microsoft Entra ID
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/tab-adaptive-cards/csharp/README.md:72]: not given in the document regarding V2, that why i kept Microsoft Entra ID

File: samples/tab-channel-group-config-page-auth/csharp/README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ urlFragment: officedev-microsoft-teams-samples-tab-channel-group-config-page-aut
 
 ## Summary
 
-There are many services that you may wish to consume inside your Teams app, and most of those services require authentication and authorization to get access to the service. Services include Facebook, Twitter, and of course Teams. Users of Teams have user profile information stored in Azure Active Directory (Azure AD) using Microsoft Graph and this article will focus on authentication using Azure AD to get access to this information.
+There are many services that you may wish to consume inside your Teams app, and most of those services require authentication and authorization to get access to the service. Services include Facebook, Twitter, and of course Teams. Users of Teams have user profile information stored in Microsoft Entra ID (Azure AD) using Microsoft Graph and this article will focus on authentication using Azure AD to get access to this information.
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/tab-channel-group-config-page-auth/csharp/README.md:20]: fixed

File: samples/tab-sso/nodejs/README.md [hunk 4]
```diff
@@ -203,8 +203,8 @@ Compared to the Hello World sample, this app has four additional routes:
...
     * If it fails (because the user hasn't granted permission to access the `profile` API), it returns an error to the `/ssoDemo` page. This error is used to display the "Consent" button which uses the Teams SDK to open the `/auth/start` page in a pop-up window.
 3. `/auth/start` and `/auth/end` routes are used if the user needs to grant further permissions. This experience happens in a seperate window. 
-    * The `/auth/start` page merely creates a valid AAD authorization endpoint and redirects to that AAD consent page.
-    * Once the user has consented to the permissions, AAD redirects the user back to `/auth/end`. This page is responsible for returning the results back to the `/ssoDemo` page by calling the `notifySuccess` API.
+    * The `/auth/start` page merely creates a valid Microsoft Entra authorization endpoint and redirects to that Microsoft Entra consent page.
+    * Once the user has consented to the permissions, Microsoft Entra redirects the user back to `/auth/end`. This page is responsible for returning the results back to the `/ssoDemo` page by calling the `notifySuccess` API.
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/tab-sso/nodejs/README.md:207]: fixed

File: samples/tab-sso/nodejs/README.md [hunk 5]
```diff
@@ -203,8 +203,8 @@ Compared to the Hello World sample, this app has four additional routes:
     * It takes the token it receives from the `/ssoDemo` page and attemps to exchange it for a new token that has elevated permissions to access the `profile` Graph API (which is usually used to retrieve the users profile photo).
     * If it fails (because the user hasn't granted permission to access the `profile` API), it returns an error to the `/ssoDemo` page. This error is used to display the "Consent" button which uses the Teams SDK to open the `/auth/start` page in a pop-up window.
 3. `/auth/start` and `/auth/end` routes are used if the user needs to grant further permissions. This experience happens in a seperate window. 
-    * The `/auth/start` page merely creates a valid AAD authorization endpoint and redirects to that AAD consent page.
-    * Once the user has consented to the permissions, AAD redirects the user back to `/auth/end`. This page is responsible for returning the results back to the `/ssoDemo` page by calling the `notifySuccess` API.
+    * The `/auth/start` page merely creates a valid Microsoft Entra authorization endpoint and redirects to that Microsoft Entra consent page.
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/tab-sso/nodejs/README.md:206]: fixed

File: samples/tab-adaptive-cards/csharp/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Pawank-MSFT) [reply samples/tab-adaptive-cards/csharp/README.md:72]: As discussed, please undo this change. It should be "Azure Active Directory V2"

[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/tab-adaptive-cards/csharp/README.md:72]: fixed

File: samples/tab-sso/nodejs/README.md [hunk 6]
```diff
@@ -217,10 +217,10 @@ This Javascript file is served from the `/msal-auth.js` page and handles the bro
 This Javascript file is served from the `/ssoDemo` page and handles most of the client-side authentication workflow. This file is broken into three main functions:
 
 1. getClientSideToken() -
-This function asks Teams for an authentication token from AAD. The token is displayed so you can try it in Postman.
+This function asks Teams for an authentication token from Microsoft Entra ID. The token is displayed so you can try it in Postman.
 
 2. getServerSideToken() -
-This function sends the token to the backend to exchange for elevated permissions using AAD's [on-behalf-of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-on-behalf-of-flow). In this case, it sends the token to the `/getProfileOnBehalfOf` route.
+This function sends the token to the backend to exchange for elevated permissions using Microsoft Entra's [on-behalf-of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-on-behalf-of-flow). In this case, it sends the token to the `/getProfileOnBehalfOf` route.
```
Comment (Pawank-MSFT) [inline samples/tab-sso/nodejs/README.md:223]: Microsoft Entra's >> Microsoft Entra ID's

**Review by Pawank-MSFT — COMMENTED** [review]
One small comments please fix.

File: samples/tab-sso/nodejs/README.md (hunk 6 above)
Comment (Mohammed-MSFT) [reply samples/tab-sso/nodejs/README.md:223]: fixed

Comment (Pawank-MSFT) [reply samples/tab-sso/nodejs/README.md:223]: Not Fixed

Comment (Mohammed-MSFT) [reply samples/tab-sso/nodejs/README.md]: Thanks, fixed now.

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**